dts/bindings/stepper/gpio-stepper-controller.yaml

```diff
@@ -0,0 +1,40 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 Carl Zeiss Meditec AG
+# SPDX-License-Identifier: Apache-2.0
+
+description: GPIO Stepper Controller cluster for darlington transistor arrays or dual H-bridge
+  stepper {
+    compatible = "gpio-steppers";
+    status = "okay";
```

Comment:
this is not really needed, normally soc peripherals are set as disabled and have to be explicitly enabled, this I imagine would live in the board file so it's just enabled all the time, just drop it, keep the example compact

Reply:
done